### PR TITLE
Tweak warning message to point to #1593, asking for help

### PR DESCRIPTION
**Pull request by GBeushausen**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ Silex, a simple Web Framework
 
 **WARNING**: Silex is in maintenance mode only. Ends of life is set to June
 2018. Read more on `Symfony's blog <http://symfony.com/blog/the-end-of-silex>`_.
+But Silex can be saved! Any **help** to keep Silex alive is greatly appreciated. See `#1593 <https://github.com/silexphp/Silex/issues/1593>`_.
+
```
Comment (fabpot) [inline README.rst]: Extra blank line should be removed.

File: README.rst [hunk 2]
```diff
@@ -3,6 +3,8 @@ Silex, a simple Web Framework
 
 **WARNING**: Silex is in maintenance mode only. Ends of life is set to June
 2018. Read more on `Symfony's blog <http://symfony.com/blog/the-end-of-silex>`_.
+But Silex can be saved! Any **help** to keep Silex alive is greatly appreciated. See `#1593 <https://github.com/silexphp/Silex/issues/1593>`_.
```
Comment (fabpot) [inline README.rst]: Can you cur at around 80 chars?

**Comment by GBeushausen** [comment]
Thanks. I've adjusted the layout.

File: README.rst [hunk 3]
```diff
@@ -2,9 +2,9 @@ Silex, a simple Web Framework
 =============================
 
 **WARNING**: Silex is in maintenance mode only. Ends of life is set to June
-2018. Read more on `Symfony's blog <http://symfony.com/blog/the-end-of-silex>`_.
-But Silex can be saved! Any **help** to keep Silex alive is greatly appreciated. See `#1593 <https://github.com/silexphp/Silex/issues/1593>`_.
-
+2018. Read more on `Symfony's blog <http://symfony.com0/blog/the-end-of-silex>`_.
```
Comment (xabbuh) [inline README.rst]: the 0 here looks wrong

**Comment by fabpot** [comment]
Thank you @GBeushausen.

**Pull request closed without merging**